src/pages/index.page.jsx

```diff
@@ -6,56 +6,69 @@ export default function Home() {
-          <div className="home-top__text-column">
-            <h1>Accessibility Cheatsheets for Components</h1>
-            <p className="home-top__top-description">
-              Intro copy goes here, such as: Build accessible components more
-              accurately and in less time with these easy-to-read cheatsheets
-              based on WCAG and ARIA best practices. Check back often as
-              we&#39;re adding new components frequently!
-            </p>
-          </div>
-          <div>
-            <img
-              className="home-top__top-image"
-              src="/home-header-image.svg"
-              alt="Illustration of dog surrounded by accessible components"
-            />
+        <div className="home-wrapper">
+          <div className="home-top">
+            <div className="home-top__text-column">
+              <h1>Accessibility Cheatsheets for Components</h1>
+              <p className="home-top__top-description">
+                Intro copy goes here, such as: Build accessible components more
+                accurately and in less time with these easy-to-read cheatsheets
+                based on WCAG and ARIA best practices. Check back often as
+                we&#39;re adding new components frequently!
+              </p>
+            </div>
+            <div>
+              <img
+                className="home-top__top-image"
+                src="/home-header-image.svg"
+                alt="Illustration of dog surrounded by accessible components"
+              />
+            </div>
           </div>
         </div>
-        <ul className="components-grid">
-          <li className="components-grid__item components-grid__item--accordion">
-            <ComponentCard
-              name="Accordion"
-              slug="/accordion"
-              image="/accordion_artwork.svg"
-            />
-          </li>
-          <li className="components-grid__item components-grid__item--dialog">
-            <ComponentCard
-              name="Dialog"
-              slug="/dialog"
-              image="/dialog_artwork.svg"
-            />
-          </li>
-          <li className="components-grid__item components-grid__item--disclosure">
-            <ComponentCard
-              name="Disclosure"
-              slug="/disclosure"
-              image="/disclosure_artwork.svg"
-            />
-          </li>
-          <li className="components-grid__item components-grid__item--tabs">
-            <ComponentCard name="Tabs" slug="/tabs" image="/tabs_artwork.svg" />
-          </li>
-          <li className="components-grid__item components-grid__item--max-scott">
-            <img
-              src="/max-grid.svg"
-              alt="Illustration of cute small white terrier dog with tongue out and head turned left looking like he is curious"
-            />
-          </li>
-        </ul>
+        <article className="components-grid">
+          <h2 className="components-grid__heading">Components</h2>
```

Comment:
This heading also has a font weight of 400 in the designs.

Reply:
Yes, I definitely just missed the font-weights. 😬 Fixed now! https://github.com/sparkbox/accessible-components/pull/58/commits/aff6b24b6e140e1a10596695338954a68242bf19

Reply:
> And this `<h2>` also needs line height added for mobile and desktop.

I'm going to go back to Jelly and ask about this, because the line-heights on desktop (228px) overlap the component cards themselves, and I have no way to measure the actual distance between the literal text and the tops of those cards. I got the mobile line-height in place though: 

https://github.com/sparkbox/accessible-components/pull/58/commits/a0d86f80b4b7ccd7aeac37d5ee1ca362433b0ab7